src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the Roboto font is properly loaded in your HTML file. The current setup in the HTML file does not correctly link to the Google Fonts stylesheet.